src/controllers/RequestController.js

```diff
@@ -66,6 +66,45 @@ class RequestController {
       return HelperMethods.serverError(res);
     }
   }
+
+  static async editRequest(req, res) {
+    try {
+      const { id } = req.decoded;
+      const {
+        body
+      } = req;
+
+      const requestExist = await Request.findOne({
+        where: {
+          id: body.requestId,
+          userId: id,
+        }
+      });
+      if (requestExist) {
+        if (requestExist.dataValues.status === 'pending') {
+          if (requestExist.dataValues.returnDate) {
+            const convertFlightDate = new Date(body.flightDate).toISOString();
+            const convertReturnDate = new Date(body.returnDate).toISOString();
+            if (convertFlightDate > convertReturnDate) return HelperMethods.clientError(res, 'Invalid Date Parameters', 400);
```

Comment:
should the message returned here be `Invalid Date Parameters` or `The flight date cannot be after the return date.`